deploy/helm/sumologic/values.yaml

```diff
@@ -4645,27 +4647,29 @@ otellogs:
             type: router
             routes:
               - output: parser-docker
-                expr: '$$body matches "^\\{"'
+                expr: 'body matches "^\\{"'
               - output: parser-crio
-                expr: '$$body matches "^[^ Z]+ "'
+                expr: 'body matches "^[^ Z]+ "'
               - output: parser-containerd
-                expr: '$$body matches "^[^ Z]+Z"'
+                expr: 'body matches "^[^ Z]+Z"'
           ## Parse CRI-O format
           - id: parser-crio
             type: regex_parser
-            regex: '^(?P<time>[^ Z]+) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*) (?P<log>.*)$'
+            regex: '^(?P<time>[^ Z]+) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*)( |)(?P<log>.*)$'
             output: merge-cri-lines
+            parse_to: body
             timestamp:
-              parse_from: time
+              parse_from: body.time
               layout_type: gotime
               layout: '2006-01-02T15:04:05.000000000-07:00'
           ## Parse CRI-Containerd format
           - id: parser-containerd
             type: regex_parser
-            regex: '^(?P<time>[^ ^Z]+Z) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*) (?P<log>.*)$'
+            regex: '^(?P<time>[^ ^Z]+Z) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*)( |)(?P<log>.*)$'
```

Comment:
I'm not sure I understand what this change is supposed to accomplish. Can you explain?

Reply:
We observed in our environments, that there were some empty logs which do not contains `<log>` part and finish just after `<logtag>` part. This change should mitigate it and prevent logging parsing errors